src/config/field-overrides/relationshipType.js

```diff
@@ -80,7 +99,28 @@ class Constraint extends Component {
         });
 
         this.setState({
-            embeddedValue: relationshipConstraint[objType],
+            embeddedValue: relationshipConstraint,
+        });
+    };
+
+    renderObjectSelect = () => {
+        const entity = this.state.relationshipEntity;
+        const modelTypes = modelTypesForRelationshipEntity[entity];
+        return modelTypes.map(objOpts => {
+            const modelType = objOpts.modelType;
+            return (
+                <DropdownAsync
+                    {...this.props}
+                    isRequired={objOpts.required}
+                    labelText={`${this.translate('select')} ${this.translate(
+                        camelCaseToUnderscores(modelType),
```

Comment:
labelText={`${this.translate('select')} ${this.translate(camelCaseToUnderscores(modelType) ${objOpts.required ? ' *' : ''}}

Reply:
This is the only change I propose for now.